src/screens/GroupInfo/elements/Header.js

```diff
@@ -37,6 +38,6 @@ const styles = StyleSheet.create({
     fontSize: normalizeFontSize(16),
     fontFamily: fonts.inter[600],
     lineHeight: normalizeFontSize(19.36),
-    color: '#000'
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hard-coded color values with references to color constants from a theme file is a good practice. It improves maintainability and consistency across the application.
```diff
-    color: '#000'
+    color: COLORS.black
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->